ingestion/src/metadata/ingestion/source/database/bigquery/metadata.py

```diff
@@ -251,6 +254,12 @@ def yield_tag(self, schema_name: str) -> Iterable[OMetaTagAndClassification]:
             logger.debug(traceback.format_exc())
             logger.warning(f"Skipping Policy Tag: {exc}")
 
+    def get_schema_description(self, schema_name: str) -> Union[str, None]:
+        query_resp = self.client.query(
+            BIGQUERY_SCHEMA_DESCRIPTION.format(schema_name=schema_name)
+        )
+        return [query_value.schema_description for query_value in query_resp][0] or ""
```

Comment:
can't we directly fetch the schema_description of first schema? instead of looping over the results?

Reply:
we could do something like this: list(a)[0].schema_description